databricks/koalas/indexes.py

```diff
@@ -386,6 +386,8 @@ def to_numpy(self, dtype=None, copy=False):
         array([0, 1, 2, 3])
         >>> ks.DataFrame({'a': ['a', 'b', 'c']}, index=[[1, 2, 3], [4, 5, 6]]).index.to_numpy()
         array([(1, 4), (2, 5), (3, 6)], dtype=object)
+        >>> ks.DataFrame({'a': ['a', 'b', 'c']}, index=[[1, 2, 3], [4, 5, 6]]).index.to_numpy(copy=True)
```

Comment:
too long here. (104 > 100)

Reply:
FYI: You can run a linter-check before push the commit via `/dev/lint-python`

Reply:
@itholic sorry I forgot to check lint